src/tutorial/microbe_editor/MicrobeEditorTutorialGUI.tscn

```diff
@@ -554,6 +555,47 @@ Description = "EDITOR_TUTORIAL_EARLY_GOAL"
 ShowDelay = 0.7
 PreventsMouseCaptureWhileOpen = true
 
+[node name="NucleusTutorial" parent="." instance=ExtResource("9")]
```

Comment:
As this doesn't have the button anymore, couldn't this use the default tutorial scene so this doesn't need a manual setup of the child nodes in here? Like the "editor tutorial early goal" tutorial.